frontend/src/helpers/Analytics.ts

```diff
@@ -147,8 +148,11 @@ export class Analytics {
     window.analytics.track(name, context)
   }
 
-  public trackConnect(name: string, data?: { id: string; name: string; typeID?: number }, error?: ISimpleError) {
-    if (data) this.track(name, { ...data, error, connectionType: CONNECTION_TYPE_FAILOVER })
+  public trackConnect(name: string, data?: { id: string; name: string; typeID?: number, isP2P?: boolean }, error?: ISimpleError) {
```

Comment:
This looks good, but I think we should also not default to "failover" if isP2P is not set. This is how I'm handling it for display. Something similar I think should be done to handle undefined in the analytics tracking:
```
                connection?.isP2P === undefined
                  ? 'None'
                  : connection?.isP2P === true
                  ? 'Peer to peer'
                  : 'Proxy failover',
```